tekton/resources/cd/catalog-template.yaml

```diff
@@ -57,7 +57,9 @@ spec:
     taskSpec:
       workspaces:
         - name: serviceaccount
+          type: string
```

Comment:
This is a bit weird since workspace doesn't have a field Type: https://github.com/tektoncd/pipeline/blob/a0c7d31e23f472cca39690410253a0739f3189db/pkg/apis/pipeline/v1beta1/task_types.go#L122